apps/src/code-studio/sectionsRedux.js

```diff
@@ -0,0 +1,49 @@
+import Immutable from 'immutable';
+import _ from 'lodash';
+
+// Action types
+export const SET_SECTIONS = 'sections/SET_SECTIONS';
+export const SELECT_SECTION = 'sections/SELECT_SECTION';
+
+// Action Creators
+export const setSections = sections => ({
+  type: SET_SECTIONS,
+  sections
+});
+
+export const selectSection = sectionId => ({
+  type: SELECT_SECTION,
+  sectionId
+});
+
+const SectionData = Immutable.Record({
+  selectedSection: null,
+  sectionsLoaded: false,
```

Comment:
While we're talking about names, I'd suggest `selectedSectionId` and `areSectionsLoaded` - at your discretion though.